### PR TITLE
Add support for documentation comments

### PR DESCRIPTION
**Pull request by Maistho**
Hi, this PR is to add support for documentation comments on the data classes.

I've added a single test for this and am unsure where to look to add more tests to make sure it is robust.

Since this is my first contribution to moor, please let me know if there's anything you'd like changed.

Thanks for making moor, it's a really useful package. :)

### CODE REVIEW

File: moor_generator/lib/src/model/column.dart [hunk 1]
```diff
@@ -96,6 +96,9 @@ class MoorColumn implements HasDeclaration, HasType {
   @override
   final UsedTypeConverter typeConverter;
 
+  /// The documentation comment associated with this column
+  final String documentationComment;
```
Comment (simolus3) [inline moor_generator/lib/src/model/column.dart:100]: The documentation should point out that each line of `documentationComment` starts with `///` already.

**Review by simolus3 — COMMENTED** [review]
Thank you for your contribution :+1: 

Overall the implementation looks really good, I just left one minor comment.

File: moor_generator/lib/src/model/column.dart (hunk 1 above)
Comment (Maistho) [reply moor_generator/lib/src/model/column.dart:102]: Agreed and updated

**Comment by Maistho** [comment]
I updated the documentation and rebased on the latest commits from the develop branch. :)

**Comment by simolus3** [comment]
Looks great, thanks!

**Pull request merged**